cmd/catchpointdump/database.go

```diff
@@ -99,14 +106,19 @@ func checkDatabase(databaseName string, outFile *os.File) error {
 
 	var stats merkletrie.Stats
 	err = dbAccessor.Atomic(func(ctx context.Context, tx *sql.Tx) (err error) {
-		committer, err := ledger.MakeMerkleCommitter(tx, false)
+		committer, err := ledger.MakeMerkleCommitter(tx, ledgerTrackerStaging)
 		if err != nil {
 			return err
 		}
 		trie, err := merkletrie.MakeTrie(committer, ledger.TrieMemoryConfig)
 		if err != nil {
 			return err
 		}
+		root, err := trie.RootHash()
+		fmt.Fprintf(outFile, " Root: %s\n", root.String())
+		if err != nil {
+			return err
+		}
```

Comment:
Does printing the root even when `err != nil` make sense?

Also, doesn't String() get called implicitly by `%s`?